app/src/main/java/org/jellyfin/mobile/controller/ServerController.kt

```diff
@@ -36,7 +37,8 @@ class ServerController(
         appPreferences.currentUserId = withContext(Dispatchers.IO) {
             userDao.upsert(serverId, userId, accessToken)
         }
-        apiClient.SetAuthenticationInfo(accessToken, userId)
+        appPreferences.currentUserUuid = UUID.fromString(userId.replace("^([a-z\\d]{8})([a-z\\d]{4})([a-z\\d]{4})([a-z\\d]{4})([a-z\\d]{12})\$".toRegex(), "$1-$2-$3-$4-$5")) // TODO Use UUIDSerializer
+        apiClient.accessToken = accessToken
```

Comment:
I think the UUID helpers from the SDK are exposed, not sure why I didn't use those here...